enclave_manager/avalon_enclave_manager/wpe/wpe_enclave_manager.py

```diff
@@ -61,15 +60,19 @@ def _create_signup_data(self):
         signup_data = enclave_info.WorkOrderProcessorEnclaveInfo(
             self._config.get("EnclaveModule"))
         self._wpe_requester = WPERequester(self._config)
+        self._worker = self._worker_kv_delegate.get_worker()
```

Comment:
The worker id should corresponds to the listener end point. WPE can add worker id as a config parameter in wpe_config.toml. We should hard code the worker id string